### DISCUSSION TITLE
CP Helfer Lohn global festlegen bzw Admin Berechtigung

### DISCUSSION THREAD

**Discussion by G0ty88** [Courseplay/Courseplay_FS22#2973] [Q&A]
Hallo, ist es möglich die CP Einstellung für den Lohn des Helfers festzulegen und diesen via Adminrechte zu sichern? Wir spielen mit CP im Multiplayer und uns ist aufgefallen, dass einige die CP kosten runtersetzen oder gar deaktivieren. Wir möchten aber das die Helfer etwas kosten. Gibt es entweder die Möglichkeit die Einstellung nur jmd mit Adminrechten festlegen zu lassen? Oder aber ist es möglich die Kosten des Helfers festzulegen und nicht veränderbar zu machen ?

**Comment by Tensuko** [comment]
Mit Leuten spielen denen man trauen kann wäre eine Option.
Es gibt keinen Plan dort eine Adminbeschränkung ein zu bauen.
